tideways.c

```diff
@@ -2532,6 +2532,7 @@ void hp_init_trace_callbacks(TSRMLS_D)
 
     cb = tw_trace_callback_event_dispatchers2;
     register_trace_callback("HookCore::coreCallHook", cb); // PrestaShop 1.6
+    register_trace_callback("Neos\\Flow\\SignalSlot\\Dispatcher::dispatch", cb);
```

Comment:
This only works as long the Dispatcher isn't overwritten by AOP right? does that happen?